CI/Dockerfile

```diff
@@ -5,7 +5,7 @@
 
 # Global ARGS set before the first build stage are accessable by all build stages
 ARG EMBREE_BRANCH='v3.6.1'
-ARG geant4_version=10.7.4
+ARG geant4_version=11.1.2
```

Comment:
not sure if we need/want to update the default yet - I'll think about it as I review the rest of the PR

Reply:
I will be testing with both